K8s-deployment/K8s-cluster/addons/ingress-controller/memcached/example-resource-values.yaml

```diff
@@ -1,11 +1,11 @@
 resources:
   limits:
-    cpu: 200m
-    memory: 256Mi
+    cpu: 50m
+    memory: 0.1Mi
```

Comment:
Hmm, the resource request and limit seems very low, please make it as before

Reply:
Sure, will do necessary changes.